src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -43,14 +44,27 @@ public Fixer()
 		public void Fix(DirectoryInfo directory)
 		{
 			WriteLine("Current directory: " + directory);
-			var xproj = GetXproj(directory);
+		    featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
+		    var generatedFileExists = featureFiles.Where(f => !File.Exists(f.FullName + ".cs")).ToList();
+
+            var xproj = GetXproj(directory);
 			var fakeCsproj = SaveFakeCsProj(directory, xproj);
 			GenerateSpecFlowGlue(directory, fakeCsproj);
 			DeleteFakeCsProj(fakeCsproj);
 			FixXunit(directory);
-		}
 
-		private void DeleteFakeCsProj(FileInfo fakeCsproj)
+            if(generatedFileExists.Any()) { 
+                generatedFileExists.ForEach(WarnNotExists);
+                WriteLine("Rebuild to make the above files discoverable");
+            }
+        }
+
+	    private void WarnNotExists(FileInfo featureFile)
+	    {
+	        WriteLine("New file generated: " + featureFile.FullName + ".cs. No tests in " + featureFile.Name + " will be discovered by dotnet test");
```

Comment:
Should use interpolation or similar. String concat this way should be avoided (not that performance should be a concern here, just as good practice).